love/src/components/OLE/Exposure/ExposureAdd.jsx

```diff
@@ -449,57 +482,46 @@ class ExposureAdd extends Component {
   }
 
   componentDidMount() {
-    ManagerInterface.getListExposureInstruments().then((data) => {
-      const registryMap = {};
-      Object.entries(data).forEach(([key, value]) => {
-        value.forEach((instrument) => {
-          if (!instrument) return;
-          registryMap[instrument] = key;
-        });
-      });
-      const instrumentsArray = Object.values(data)
-        .map((arr) => arr[0])
-        .filter((instrument) => instrument);
-      this.setState({
-        instruments: instrumentsArray,
-        selectedInstrument: instrumentsArray[0],
-        registryMap: registryMap,
-      });
-    });
-    ManagerInterface.getListImageTags().then((data) => {
-      this.setState({
-        imageTags: data.map((tag) => ({ name: tag.label, id: tag.key })),
-      });
-    });
+    // If exposure is not empty, then we are adding a log
```

Comment:
Apart from using an active voice, please do not refer to `we`. Instead of `we are adding` just write `add`. This also applies to two other code comments in this PR. I leave it up to you if you want to modify this now or not.